colossalai/communication/__init__.py

```diff
@@ -6,6 +6,7 @@
                   recv_backward)
 from .ring import ring_forward
 from .utils import send_tensor_meta, recv_tensor_meta
+from .profiler import (CCPROF, prof_init)
```

Comment:
Can you shift this to `colossalai.utils.profile` so that memory and bandwidth profiling can be put in one module?

Reply:
DONE

Reply:
I mean `colossalai.utils.profile` should be a module, i.e. `colossalai.utils.profile.comm_profiler.py`, so that memory profiler can be put together as well.

Reply:
Done